abp_io/AbpIoLocalization/AbpIoLocalization/Www/Localization/Resources/tr.json

```diff
@@ -174,6 +174,8 @@
     "CreateProjectWizard": "Bu sihirbaz, projenize atlamak için uygun şekilde yapılandırılmış başlangıç şablonundan yeni bir proje oluşturur.",
     "TieredOption": "Web ve Http API katmanlarının fiziksel olarak ayrıldığı katmanlı bir çözüm oluşturur. İşaretlenmezse, daha az karmaşık olan ve çoğu senaryo için uygun olan katmanlı bir çözüm oluşturur.",
     "SeparateIdentityServerOption": "Sunucu tarafını iki uygulamaya ayırır: Birincisi kimlik sunucusu için, ikincisi ise sunucu tarafı HTTP API'niz içindir.",
+    "ProgressiveWebApplicationOption": "Projeyi Progresif Web Uygulaması olarak ayarlayın.",
+    "ProgressiveWebApplication": "Progresif Web Uygulaması",
```

Comment:
I'm not sure to localize the original name "Progressive Web Application".

Google uses this localization but I'm not sure if it's clear or not. (https://support.google.com/chrome/answer/9658361?hl=tr&co=GENIE.Platform%3DDesktop)

Reply:
I think we should go with its original name.